tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/ServiceStackRedisTests.cs

```diff
@@ -25,6 +25,7 @@ public ServiceStackRedisTests(ITestOutputHelper output)
             : base("ServiceStack.Redis", output)
         {
             SetServiceVersion("1.0.0");
+            EnableDebugMode();
```

Comment:
Why is it needed?

Reply:
That was to get more log output for the `native`/`loader`/`managed`, I don't think it is needed anymore, so I'll remove it

I do see other instances of it in the repo, but I'm just gonna remove it
0a272bf9fc063ea3694850b31dbd1f0fe8999109